pdq/php/pdqhashertest.php

```diff
@@ -14,13 +14,18 @@
 //$show_timings = true;
```

Comment:
Hmm, this doesn't appear to be a a unittest in the normal sense of the word.

@schatten - Is what we did for wasm in #1010 viable here?